### PR TITLE
METAL-944: Install ironic-lib from source

### PR DESCRIPTION
**Pull request by elfosardo**

### CODE REVIEW

**Comment by elfosardo** [comment]
/override ci/prow/prevalidation-e2e-metal-ipi-virtualmedia-prevalidation
production has passed so we're good

**Comment by elfosardo** [comment]
/hold
let's wait for after the weekend

File: main-packages-list.ocp [hunk 1]
```diff
@@ -71,6 +70,7 @@ python3-sushy >= 4.7.1-0.20240118192507.435e7d1.el9
 python3-sushy-oem-idrac >= 5.0.0-0.20231218155726.da9a0e4.el9
 python3-tooz >= 4.2.0-0.20231214201931.bed303e.el9
 python3-werkzeug >= 2.2.3-2.el9
+python3-zeroconf >= 0.24.4-2.el9
```
Comment (dtantsur) [inline main-packages-list.ocp:73]: Side note: consider dropping this

**Pull request merged**